lib/agent/actions/wipe/windows.js

```diff
@@ -2,8 +2,55 @@ var os   = require('os'),
     join = require('path').join,
     exec = require('child_process').exec;
 
+var regispath = {
+  outlook_version: join('HKEY_CLASSES_ROOT', 'Outlook.Application', 'CurVEr'),
+  profileRegistry: join('HKEY_CURRENT_USER', 'Software', 'Microsoft'),
+  firefox:         join('HKEY_CLASSES_ROOT', 'FirefoxHTML', 'DefaultIcon'),
+  thunderbird:     join('HKEY_CLASSES_ROOT', 'ThunderbirdEML', 'DefaultIcon')
+};
+
+var registry = {
```

Comment:
describe the object in its name, such as `registryManager` or `registryOperations`